kernel/kernel-defaults/src/test/resources/workload_specs/basic_append/specs/write_appends/commit_2_adds.json

```diff
@@ -0,0 +1,16 @@
+{
+  "adds": [
+    {
+      "path": "dummy_data_b.parquet",
+      "size": 1024,
+      "modification_time": 1712091404545,
+      "stats": "{\"numRecords\":10,\"minValues\":{\"letter\":\"a\",\"number\":1,\"a_float\":1.1},\"maxValues\":{\"letter\":\"j\",\"number\":10,\"a_float\":10.10},\"nullCount\":{\"letter\":0,\"number\":0,\"a_float\":0}}"
+    },
+    {
+      "path": "dummy_data_c.parquet",
+      "size": 927,
+      "modification_time": 1712091405000,
+      "stats": "{\"numRecords\":5,\"minValues\":{\"letter\":\"b\",\"number\":2,\"a_float\":1.3},\"maxValues\":{\"letter\":\"j\",\"number\":10,\"a_float\":10.10},\"nullCount\":{\"letter\":0,\"number\":0,\"a_float\":0}}"
+    }
+  ]
```

Comment:
Note: we don't format it in Delta's usual  `{"add": {...}}` because I want to have a clear separation between what the engine is expected to provide and what kernel writes to the log. This forces runner implementations to go through the normal write procedure (example: `generate_append_actions` in java and `add_files` in rust)